core/pkg/model/pb/messages.pb.go

```diff
@@ -0,0 +1,876 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.32.0
+// 	protoc        v4.25.3
+// source: protos/customcost/messages.proto
+
+package pb
+
+import (
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type CustomCostResponse struct {
```

Comment:
If we're going to go to the effort of defining a nice proto schema for this data, why not just eliminate the other `CustomCostResponse` type entirely? Or is that an antipattern?

Reply:
Sean and I discussed this very thing. I was originally against it, but maybe that is worth revisiting

Reply:
That is my position for the most part, There may be some value to having the other definition for data manipulation in opencost, but we could also just extend the proto struct with any helper functions we want. For kubecost, I want the transition to duck to come from the plugin as protobuf directly and go directly to duck schema and into the db